src/AppInstallerCommonCore/Public/AppInstallerTelemetry.h

```diff
@@ -119,6 +125,9 @@ namespace AppInstaller::Logging
 
         bool IsTelemetryEnabled() const noexcept;
 
+        // Check for valid Json string and return a valid Json
+        std::wstring GetTelemetryCorelationJsonW() const;

```

Comment:
I still think this would be better as returning a `const std::wstring&`

Reply:
Ok. Yes.